CoreTweet.Shared/Labs/V2/Media.cs

```diff
@@ -0,0 +1,199 @@
+        public int Playback100Count { get; set; }
+        // TODO: is it ok to be `int`?
+    }
+
+    public class MediaPublicMetrics : CoreBase
+    {
+        /// <summary>
+        /// The count of how many times the video attached to this Tweet has been viewed. This is the number of video views aggregated across all Tweets in which the given video has been posted. That means that the metric includes the combined views from any instance where the video has been Retweeted or reposted in separate Tweets. This returns the total count of video views from both organic and paid contexts, in order to maintain consistency with the counts shown publicly on Twitter.
+        /// </summary>
+        [JsonProperty("view_count")]
+        public int ViewCount { get; set; }
+        // TODO: is it ok to be `int`?
+    }
+
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum MediaType
+    {
+        [EnumMember(Value = "animated_gif")]
+        AnimatedGif,
+        [EnumMember(Value = "photo")]
+        Photo,
+        [EnumMember(Value = "video")]
+        Video,
+    }
+
+    /// <summary>
+    /// List of media fields to return in the Tweet media object. The response will contain the selected fields only if a Tweet contains media attachments.
+    /// </summary>
+    [Flags]
+    public enum MediaFields
+    {
+        None             = 0x00000000,
+        DurationMs       = 0x00000001,
+        Height           = 0x00000002,
+        MediaKey         = 0x00000004,
+        NonPublicMetrics = 0x00000008,
+        PreviewImageUrl  = 0x00000010,
+        PublicMetrics    = 0x00000020,
+        Type             = 0x00000040,
+        Url              = 0x00000080,
+        Width            = 0x00000100,
+    }
+
+    public static class MediaFieldsExtensions
+    {
+        public static string ToQueryString(this MediaFields value)
+        {
+            if (value == MediaFields.None)
+                return "";
+
+            var builder = new StringBuilder();
+
+            if ((value & MediaFields.DurationMs) != 0)
+                builder.Append(",duration_ms");
+            if ((value & MediaFields.Height) != 0)
+                builder.Append(",height");
+            if ((value & MediaFields.MediaKey) != 0)
+                builder.Append(",media_key");
+            if ((value & MediaFields.NonPublicMetrics) != 0)
+                builder.Append(",non_public_metrics");
+            if ((value & MediaFields.PreviewImageUrl) != 0)
+                builder.Append(",preview_image_url");
+            if ((value & MediaFields.PublicMetrics) != 0)
+                builder.Append(",public_metrics");
+            if ((value & MediaFields.Type) != 0)
+                builder.Append(",type");
+            if ((value & MediaFields.Url) != 0)
+                builder.Append(",url");
+            if ((value & MediaFields.Width) != 0)
+                builder.Append(",width");
+
+            return builder.ToString().Substring(1);
```

Comment:
To avoid reallocation, you can write `build.Append("hoge,")` and finally `builder.Remove(builder.Length - 1, 1)`.